modules/msk/neoantigenutils/neoantigeninput/environment.yml

```diff
@@ -1,9 +1,7 @@
 ---
 # yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/environment-schema.json
-name: "neoantigenutils_neoantigeninput"
 channels:
   - conda-forge
   - bioconda
-  - defaults
 dependencies:
-  - "NEOANTIGENUTILS"
+  - "YOUR-TOOL=HERE"
```

Comment:
Why was it changed?

Reply:
Looks like dependencies should have the link to the bioconda package, https://github.com/nf-core/modules/blob/master/modules/nf-core/vcf2maf/environment.yml. Since we really don't link those packages in we could leave it as is:

The alternative option we can do to match the schema is TOOL_NAME=NA

Reply:
Ah okay so this can be anything if theres no specific bioconda package and theres an "=" there.  I'm okay with whatever here then.

Reply:
Sounds good.  Also, I think we should move away from anaconda and the defaults conda channel as they recently changed their licensing for research purposes and now any organization with 200 or more employees or contractors, including research and academic organizations, are now required to purchase a paid license to use Anaconda's software, including packages on the defaults channel.